Titanium.Web.Proxy/Network/CertificateManager.cs

```diff
@@ -110,6 +140,18 @@ internal X509Certificate2 RootCertificate
             }
         }
 
+
+        private string password_rootCert = string.Empty;
```

Comment:
Check the naming conventions. Your code contains a lot of names (field, property, method) which has incorrect names. Please fix all of them.